deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -50,15 +50,17 @@ function find_cni_conf_file() {
 }
 
 function check_install() {
-  cfgfile_nm=$(find_cni_conf_file)
-  if [[ "${cfgfile_nm}" != "${CNI_CONF_NAME}" ]]; then
-    if [[ "${CNI_CONF_NAME_OVERRIDE}" != "" ]]; then
-       # Install was run with overridden cni config file so don't error out on the preempt check.
-       # Likely the only use for this is testing this script.
-       echo "WARNING: Configured CNI config file \"${MOUNTED_CNI_NET_DIR}/${CNI_CONF_NAME}\" preempted by \"$cfgfile_nm\"."
-    else
-       echo "ERROR: CNI config file \"${MOUNTED_CNI_NET_DIR}/${CNI_CONF_NAME}\" preempted by \"$cfgfile_nm\"."
-       exit 1
+  if [ "${CHAINED_CNI_PLUGIN}" != "true" ]; then
```

Comment:
I think you mean `==`.

Anyway, I think it's better check if it's `!= true` and do an early exit. It's a less intrusive change.

Reply:
OMG, you're right. Thanks!!!

Yeah, I usually use early exists, but in this case, only the first part of the function needs to be skipped.